Source/Common/Robot.py

```diff
@@ -12,33 +12,40 @@
 # The whiskers have a diameter, and a position relative to the robot they are attached to
 class Whisker:
     # Constructor
-    # x_pos: The x position of the whisker
-    # y_pos: The y position of the whisker
+    # pos: a Vec2 representing the position of the whisker relative to the center
     # diameter: The diameter of the whisker
-    def __init__(self, x_pos, y_pos, diameter):
-        self.x_pos = x_pos
-        self.y_pos = y_pos
+    def __init__(self, pos: Vec2, diameter: float):
+        self.pos = (pos,)
```

Comment:
I looked it up. Did you mean to make this a singleton? It looks like it would probably work because of how robust your Vec2 is, but it seems unnecessary.